docs/add-root-ca.rst

```diff
@@ -0,0 +1,45 @@
+Add a root CA to the web browser
+================================
+
+.. todo::
+
+  The following test has been realized on vagrant only - One should list
+  all requirements for reaching the UI using an operationnal cluster.
+  Web browsers version used for tests:
+  Googe Chrome: Version 75.0.3770.80 | Mozilla Firefox: Version 67.0
+
+Run the following command as ``root`` on the bootstrap node:
+Copy the root CA from the bootstrap on your host:
+
+.. code-block:: shell
+
+  $ scp /etc/kubernetes/pki/ca.crt <user>@<IP>:<directory>
+
+Then go back on you host and open you web browser ton install the root CA.
+
+Install on Google Chrome
+------------------------
+
+Navigate on ``chrome://settings``
```

Comment:
Since this is a list of instruction to follow sequentially, wouldn't it make sense to format it as a bullet list?

Reply:
I tried to do a list, please tell me if it's better like this.

Reply:
It works!